src/layout.js

```diff
@@ -208,9 +208,12 @@ export function getLengthUnits(length) {
 /**
  * Returns the numeric value of a CSS length value.
  * @param {!LengthDef|string} length
- * @return {number}
+ * @return {number|undefined}
  */
 export function getLengthNumeral(length) {
+  if (!length) {
```

Comment:
I think we probably have to work with the result, since the `length` could be something weird like `auto`. E.g.
```
const res = parseFloat(length);
return Number.isNaN(res) ? undefined : res;
```

And that would need a test.